spark/v3.5/spark-extensions/src/test/java/org/apache/iceberg/spark/extensions/TestReplaceBranch.java

```diff
@@ -149,9 +148,6 @@ public void testReplaceBranchWithRetain() throws NoSuchTableException {
     long first = table.currentSnapshot().snapshotId();
     String branchName = "b1";
     table.manageSnapshots().createBranch(branchName, first).commit();
-    SnapshotRef b1 = table.refs().get(branchName);
-    Integer minSnapshotsToKeep = b1.minSnapshotsToKeep();
-    Long maxSnapshotAgeMs = b1.maxSnapshotAgeMs();
```

Comment:
why are these being removed? I think those need to stay

Reply:
The sql in this test is like `ALTER TABLE catalog.db.tbl REPLACE BRANCH branch AS OF VERSION snapshotId RETAIN maxRefAge timeUnit`. For this query, the `minSnapshotsToKeep` and `maxSnapshotAgeMs` are always null in the current impl. If the query has `WITH SNAPSHOT RETENTION` the parameters are filled. So I remove this part and change the tests to null checks.

Reply:
ok thanks for confirming, no need to change anything as I think being explicit and making sure those are null is good in this case

Reply:
thank you!